src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/GetAllocationResourceIntTest.kt

```diff
@@ -241,6 +244,12 @@ class GetAllocationResourceIntTest : IntegrationTestBase() {
               courseAllocation(courseActivity = courseActivity, startDate = "$today", suspended = true)
             }
           }
+          offender {
+            booking {
+              incentive(iepLevelCode = "BAS")
+              courseAllocation(courseActivity = courseActivity, startDate = "$today")
+            }
+          }
```

Comment:
Add a missing pay band allocation test case to each of the allocation selection tests to make sure that we only include allocations that are in the migration.